docs/source/install/ewc_ha.rst

```diff
@@ -0,0 +1,279 @@
+
+.. figure :: /_static/images/helm-chart-notes.png
+    :align: center
+
+
+The installation uses some unsafe defaults which are recommended to change thoughtfully for production use via Helm ``values.yaml``.
+
+Helm values.yaml
+________________
+Helm package ``stackstorm-enterprise-ha`` comes with the default settings in ``values.yaml``.
+Fine-tune them to achieve desired configuration for the StackStorm Enterprise HA K8s cluster.
+
+You can configure:
+
+- number of replicas for each component
+- st2 auth secrets
+- st2.conf settings
+- RBAC roles, assignments and mappings
+- custom st2 pack configs
+- st2web SSL certificate
+- SSH private key
+- K8s resources and settings to control pod/deployment placement
+- configuration for Mongo, RabbitMQ clusters
+
+.. note::
+
+    **NB!** It's highly recommended to set your own secrets as file contains unsafe defaults like self-signed SSL certificates, SSH keys, StackStorm access credentials and MongoDB/RabbitMQ passwords!
+
+Upgrading
+_________
+Once you make any changes to Helm values, upgrade the cluster:
+
+.. code-block:: bash
+
+  helm repo update
+  helm upgrade <release-name> stackstorm/stackstorm-enterprise-ha
+
+It will redeploy components which were affected by the change, taking care about keeping
+desired number of replicas to sustain every service alive during the rolling upgrade.
+
+
+Tips & Tricks
+_____________
+Save custom Helm values you want to override in a separated file, upgrade the cluster:
+
+.. code-block:: bash
+
+  helm upgrade -f custom_values.yaml <release-name> stackstorm/stackstorm-enterprise-ha
+
+Get all logs for entire StackStorm cluster with dependent services for Helm release:
+
+.. code-block:: bash
+
+  kubectl logs -l release=<release-name>
+
+Grab all logs only for stackstorm backend services, excluding st2web and DB/MQ/etcd:
+
+.. code-block:: bash
+
+  kubectl logs -l release=<release-name>,tier=backend
+
+
+Custom st2 packs
+----------------
+To follow the stateless model, shipping custom st2 packs is now part of the deployment process.
+It means that ``st2 pack install`` won't work in a distributed environment and you have to bundle all the
+required packs into a Docker image that you could codify, version and distribute via preferred Docker registry,
+is it private or not. The responsibility of such Docker image is to hold pack content and their virtualenvs.
+So custom st2 pack Docker image you have to build is just read-only dirs that are shared with the corresponding
+st2 services in a cluster.
+
+For the convenience we created new ``st2-pack-install <pack1> <pack2> <pack3>`` command
+that'll help to install custom packs during the Docker build process without relying on DB and MQ connection.
+
+Helm chart brings helpers to simplify this experience like `stackstorm/st2pack-builder <https://hub.docker.com/r/stackstorm/st2packs-builder/>`_
```

Comment:
https://hub.docker.com/r/stackstorm/st2packs:builder and https://hub.docker.com/r/stackstorm/st2packs:runtime

However, the user will build st2packs image using the simple `Dockerfile` in `stackstorm-enterprise-ha` git repo.